ui/app/components/ens-input.js

```diff
@@ -85,14 +85,6 @@ EnsInput.prototype.lookupEnsName = function () {
   const recipient = document.querySelector('input[name="address"]').value
   const { ensResolution } = this.state
 
-  if (!this.ens) {
-    return this.setState({
-      loadingEns: false,
-      ensFailure: true,
-      hoverText: 'ENS is not supported on your current network.',
-    })
-  }
```

Comment:
We don't ever hit this part of the code if the network does not support ENS, due to the conditional on the `mount` function.